setup.py

```diff
@@ -37,7 +37,7 @@
 
```

Comment:
@domfournier, I would be curious to know which tools requires `setup.py`

Using Poetry is supposed to save the hassle of having to maintain `setup.py`, `requirements.txt`, `setup.cfg`, `MANIFEST.in`, `Pipfile`, …

Reply:
I can move the `environment.yml` to the docs, and we just kill the setup.py?